pkg/config/config_test.go

```diff
@@ -729,9 +617,9 @@ func TestNewOrgConfigGroup(t *testing.T) {
 		}
 	},
 	"values": {
-		"Endpoints": {
+		"AnchorPeers": {
 			"mod_policy": "Admins",
-			"value": "Cg8xMjMuNDUuNjc3OjgwODA=",
+			"value": "CgkKBWhvc3QxEHs=",
```

Comment:
Unrelated: Shouldn't this be getting handled by protolator? Do we need to open a bug for that?